src/components/PbBookCover.vue

```diff
@@ -1,9 +1,12 @@
 <template>
-  <img
-    v-img-fallback="'/assets/images/default-book-cover.jpg'"
-    class="w-full"
-    :src="image"
+  <div
+    v-lazy-container="{ selector: 'img', error: defaulImg, loading: defaulImg }"
```

Comment:
@fdalcin regarding your question, the error property does the job. I will remove the fallback lib dependency, you're right we won't need it anymore

Reply:
We just need to replace the library on the collection cover I think

Reply:
the collection cover doen't contain any fallback img

Reply:
Yes, I saw that after that's why I deleted the message.